sys/docker/alpine-nginx/start.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/sh
+# vim:sw=4:ts=4:et
+
+IS_DEV=$(ping -c 1 -t 5 169.254.169.253 > /dev/null; echo $?)
+
+# Only for production
+if [ "$IS_DEV" == "0" ]; then
+    /opt/aws/amazon-cloudwatch-agent/bin/start-amazon-cloudwatch-agent
+fi
+
+set -e
+
+if [ -z "${NGINX_ENTRYPOINT_QUIET_LOGS:-}" ]; then
+    exec 3>&1
+else
+    exec 3>/dev/null
+fi
+
+if [ "$1" = "nginx" -o "$1" = "nginx-debug" ]; then
+    if /usr/bin/find "/docker-entrypoint.d/" -mindepth 1 -maxdepth 1 -type f -print -quit 2>/dev/null | read v; then
```

Comment:
this will break if a file has spaces .. `-print0`

Reply:
I've copied this file from the nginx distro. I'd like to maintain a match 1:1.